cpu/sam0_common/periph/spi.c

```diff
@@ -92,9 +92,10 @@ void spi_init_pins(spi_t bus)
     gpio_init(spi_config[bus].miso_pin, GPIO_IN_PD);
     gpio_init(spi_config[bus].mosi_pin, GPIO_OUT);
     gpio_init(spi_config[bus].clk_pin, GPIO_OUT);
-    gpio_init_mux(spi_config[bus].miso_pin, spi_config[bus].miso_mux);
+    /* The out pins will be muxed during acquire and release.
+     * Otherwise these pins will get HIGH-Z during deep sleep of the MCU,
+     * which may leads to unexpected current consumption by SPI slaves. */
```

Comment:
MOSI doesn't have this issue?

Reply:
Whops. I swapped them! Input pins aren't problematic. Output pins are.
Thank you for the catch!